components/pinscan/pinscan.cpp

```diff
@@ -50,6 +50,41 @@ static const int PIN_MAP[] = {
     PIN_PA30,  // PA30 / UART2_TX / I2C0_SDA / PWM4
 
 };
+#elif defined(USE_ESP8266)
+static const int PIN_MAP[] = {
+    D0, // GPIO16
+    D1, // GPIO5 / SCL
+    D2, // GPIO4
+    D3, // GPIO0
+    D4, // GPIO2 / SDA
+    D5, // GPIO14 / SCK
+    D6, // GPIO12 / MISO
+    D7, // GPIO13 / MOSI
+    D8, // GPIO15 / SS
+    D9, // GPIO3
+    D10, // GPIO1
+};
+#elif defined(USE_ESP32)
+static const int PIN_MAP[] = {
+    T1,     // GPIO0
+    T2,     // GPIO2
+    T0,     // GPIO4
```

Comment:
What are the T# pins and where are they defined? Just like on esp8266, you can simply use numeric values here, and describe pin functionality in the comment (and in YAML).